modules/hal_nxp/mcux/mcux-sdk-ng/drivers/drivers.cmake

```diff
@@ -176,7 +176,7 @@ elseif((${MCUX_DEVICE} MATCHES "MK(28|66)") OR (${MCUX_DEVICE} MATCHES "MKE(14|1
 endif()
   set_variable_ifdef(CONFIG_HAS_MCUX_XCACHE		CONFIG_MCUX_COMPONENT_driver.cache_xcache)
 
-if((${MCUX_DEVICE} MATCHES "MIMX9596") OR (${MCUX_DEVICE} MATCHES "MIMX8UD7") OR (${MCUX_DEVICE} MATCHES "MIMXRT118"))
+if((${MCUX_DEVICE} MATCHES "MIMX9596") OR (${MCUX_DEVICE} MATCHES "MIMX8UD7") OR (${MCUX_DEVICE} MATCHES "MIMXRT118") OR (${MCUX_DEVICE} MATCHES "MIMX94398"))
```

Comment:
why does this not use the Kconfig symbols for these devices?

Reply:
Actually the MCUX_DEVICE is upper string of CONFIG_SOC, the original code comes from hal, and it continue to use this variable,  so I don't think we need to update it to CONFIG_SOC, right? thanks.

Reply:
it's easier and shorter to compare a symbol part of a symbol against a string, would far prefer this be using the symbol, which would also highlight errors with typos because the symbols would be checked, this could have a typo in and you would have no idea

Reply:
I see, have updated, but I only updated the symbol for new added imx943 because I am not sure the old platforms are feasible to change to use config items or not, for example,  ${MCUX_DEVICE} MATCHES "MIMXRT118") will match a serial of RT118 SoC, such as rt1180 rt1190, so keep this format is better to change to use CONFIG_SOC_RT1180 OR CONFIG_SOC_RT1189.